climada/test/test_plot.py

```diff
@@ -24,15 +24,25 @@
 import pandas as pd
 import contextily as ctx
 import urllib
+import copy
 
-from climada.entity.entity_def import Entity
 from climada.hazard.base import Hazard
 from climada.hazard import Centroids
+<<<<<<< HEAD
+from climada.entity.entity_def import Entity
+from climada.entity.exposures.base import Exposures
+from climada.entity import DiscRates,ImpfTropCyclone
+from climada.entity.measures import Measure, MeasureSet
+=======
 from climada.entity.exposures.base import Exposures
 from climada.entity import DiscRates
+>>>>>>> develop
```

Comment:
Oh yes, my bad, I must have forgotten them. I made a new commit that should fix the left overs and convert docstring comments to regular comments with `#`. I hardly see comments on the code, should I also not comment ? I just thought a few comments  might be helpful in structuring the long code to create a `costbenefit` object.

Reply:
Thank you! Generally, adding comments and using them to structure your code is important and recommended. However, there is a tradeoff when it comes to adding comments where there are no changes related to the PR. If you already worked through the class and think the code might need more comments, I think it would be best to add them in a separate PR.

Reply:
Copy that! I was referring mainly to the comments in the code that I added so I won't need to open a new PR to comment the class in general, but I will keep that in mind thanks!